src/graph/test/FetchVerticesTest.cpp

```diff
@@ -318,6 +318,33 @@ TEST_F(FetchVerticesTest, NonExistVertex) {
         ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
         ASSERT_EQ(nullptr, resp.get_rows());
     }
+    {
+        cpp2::ExecutionResponse resp;
+        auto *fmt = "FETCH PROP ON * %ld";
+        auto query = folly::stringPrintf(fmt, nonExistPlayerID);
+        auto code = client_->execute(query, resp);
+        ASSERT_EQ(cpp2::ErrorCode::SUCCEEDED, code);
+        ASSERT_EQ(nullptr, resp.get_rows());
+    }
+}
+
+TEST_F(FetchVerticesTest, FetchAll) {
```

Comment:
add ut in data_test which has multi-tag in vertex

Reply:
Working on it.